rails-tips/README.md

```diff
@@ -22,3 +22,15 @@ Rails tips are fun tid-bits of info that aren't best practices or opinionated.
     end
   end
   ```
+
+##### Decorators
+Decorators should be used to abstract object presentation away from models, controllers, or views (or any other intermediary layer). Objects can be "decorated", which adds behavior to the instance.
+
+*Do use decorators for*
+- Things like `@user.first_name` (object presentation)
+- Things where object manipulation is repeated like `name.gsub("-", " ").titleize` in views or date formatting.
+
+*Don't use decorators for*
+- Model logic
+- Complex view logic (Facades are better patterns for that)
+- *Anything* that tries to modify the actual object (ie. saving/updating)
```

Comment:
It may be nice to provide an example of how we want to call decorators here.. I.e `.decorate` where possible otherwise we defer to like: `PostDecorator.decorate_collection(@posts)` (see our reports controllers for a real example)

Reply:
I'm not sure this is really necessary. There's not a great example of how we _should_ currently do things, so referring to draper documentation is probably best in those cases.